### PR TITLE
Simple per-user instance quota checking in the create call. PEP8 fixes.

### PR DESCRIPTION
**Pull request by ed-**
create_instance will 413 you if you already have as many instances as configured in the conf file.
Tests to follow in RDLI

### CODE REVIEW

**Comment by cp16net** [comment]
rebase

File: reddwarf/instance/service.py [hunk 1]
```diff
@@ -242,9 +244,22 @@ def create(self, req, body, tenant_id):
             try:
                 volume_size = int(body['instance']['volume']['size'])
             except ValueError as e:
-               raise exception.BadValue(msg=e)
+                raise exception.BadValue(msg=e)
         else:
             volume_size = None
+
+        #TODO(ed-) HELLO. THIS IS A HUGE STOPGAP.
```
Comment (hub-cap) [inline reddwarf/instance/service.py]: hehe

File: reddwarf/instance/service.py [hunk 2]
```diff
@@ -242,9 +244,22 @@ def create(self, req, body, tenant_id):
...
+        # FIRST, the quota needs to be enforced outside of create. SECOND,
+        # filtering on the task description isn't the right way to do it.
+        instance_max = int(config.Config.get('max_instances_per_user', 1))
+        number_instances = models.DBInstance.count_extant_instances(tenant_id)
+
+        if number_instances >= instance_max:
+            # That's too many, pal. Got to cut you off.
+            LOG.error(_("New instance would exceed user instance quota."))
+            msg = "User instance quota of %d would be exceeded."
```
Comment (hub-cap) [inline reddwarf/instance/service.py:261]: Why not add this to the error itself and pass that param as a kwargs?

File: reddwarf/instance/service.py [hunk 3]
```diff
@@ -284,18 +299,19 @@ def _validate_volume_size(size):
                    "integer value, %s cannot be accepted."
                    % volume_size)
             raise exception.ReddwarfError(msg)
-        max_size = int(config.Config.get('max_accepted_volume_size',
-                                         1))
+        max_size = int(config.Config.get('max_accepted_volume_size', 1))
```
Comment (cp16net) [inline reddwarf/instance/service.py:303]: should this really be set to 1 by default? maybe 10 sounds better

File: reddwarf/instance/service.py [hunk 4]
```diff
@@ -284,18 +299,19 @@ def _validate_volume_size(size):
...
         if int(volume_size) > max_size:
             msg = ("Volume 'size' cannot exceed maximum "
                    "of %d Gb, %s cannot be accepted."
                    % (max_size, volume_size))
-            raise exception.ReddwarfError(msg)
+            #raise exception.ReddwarfError(msg)
```
Comment (cp16net) [inline reddwarf/instance/service.py:308]: commented out? just remove if not needed?

Comment (hub-cap) [inline reddwarf/instance/service.py:308]: comment?

File: rsdns/client/future.py [hunk 5]
```diff
@@ -42,7 +44,8 @@ def poll(self):
                 resp_list = body['response'][self.response_list_name()]
                 self.result = self.manager.create_from_list(resp_list)
                 #self.resource_class(self, res) for res in list]
-                #self.result = Domain(self.manager, body['self.convert_callback(resp, body)
+                #self.result = Domain(self.manager,
+                #   body['self.convert_callback(resp, body)
```
Comment (hub-cap) [inline rsdns/client/future.py:48]: need these?

Comment (ed-) [inline rsdns/client/future.py:48]: Need them or not, 80 column rule

File: reddwarf/instance/service.py (hunk 4 above)
Comment (hub-cap) [inline reddwarf/instance/service.py:308]: we mustve made this at the same time :P

File: rsdns/client/future.py (hunk 5 above)
Comment (hub-cap) [inline rsdns/client/future.py:48]: understandable. nuke them tho

**Pull request closed without merging**